script/testing/microbench/micro_benchmarks_runner.py

```diff
@@ -142,7 +143,11 @@ def _execute_benchmark(self, cmd):
         LOG.debug(f'Executing command [num_threads={self.config.num_threads}]: {cmd}')
         try:
             output = subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True)
-            pretty_format_json = json.dumps(json.loads(output.decode('utf8').replace("'", '"')), indent=4)
+            
+            # Strip DBMS debug messages
+            output = re.sub(r"\[.*?\] \[.*?\] \[[\w]+\] .*\n", "", output.decode('utf8'))
```

Comment:
Technically, this might capture and eat something like `[[1] [2] [3] [4]]` which could appear in:
```
{
  'blah': ...,
  'raw_data': [[1] [2] [3] [4]],
  ...
}
```

But I ran a couple of quick tests against numpy and it doesn't seem to print that, so this is probably fine.

Reply:
I can make it specific to dates if you want

Reply:
I think it is probably fine, whoever adds more stuff will figure out that something is up when they realize their output is disappearing / it won't be flaky